.gitmodules

```diff
@@ -1,3 +1,4 @@
-[submodule "ccd-docker"]
-	path = ccd-docker
-	url = https://github.com/hmcts/ccd-docker
+[submodule "fpla-docker"]
+	path = fpla-docker
+	url = https://github.com/hmcts/fpla-docker
+	branch = dev/FPL-994-test
```

Comment:
temporary as it  depends on https://github.com/hmcts/fpla-docker/pull/10. Once merged it should be master